scanner/src/main/kotlin/PathScanner.kt

```diff
@@ -138,7 +138,10 @@ abstract class PathScanner(
 
         remainingPackages.removeAll { it in resultsFromStorage.keys }
 
-        log.info { "Scanning ${remainingPackages.size} package(s) for which no stored scan results were found." }
+        log.info {
+            "Scanning ${remainingPackages.size} package(s) for which no stored scan results were found out of total " +
+                    "${packages.size} package(s)."
```

Comment:
"... out of **a total of** x package(s)."

Reply:
The sentence sounds a bit clumsy, how about changing the log above instead to "Found stored scan results for x of y packages matching $scannerCriteria", which would also be more correct, as technically `resultsFromStorage.size` is the number of packages for which results were found, not the number of scan results in the map.

Reply:
Wuold you then remove the

    log.info { "Found ${resultsFromStorage.size} stored scan result(s) matching $scannerCriteria." }

from a few lines above?

Reply:
Resolved offline.